1-components/2-footer/src/app/footer/footer.component.ts

```diff
@@ -5,4 +5,8 @@ import { Component, ChangeDetectionStrategy } from '@angular/core';
 	templateUrl: './footer.component.html',
 	changeDetection: ChangeDetectionStrategy.OnPush,
 })
-export class FooterComponent {}
+export class FooterComponent {
+	public author = 'Angular Course';
```

Comment:
Поле является постоянным и не изменяемым. Давай подобные поля помечать `readonly`
```suggestion
	public readonly author = 'Angular Course';
```